homeassistant/components/light/lutron_caseta.py

```diff
@@ -0,0 +1,105 @@
+"""Support for Lutron Caseta lights."""
+import logging
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, SUPPORT_BRIGHTNESS, Light)
+from homeassistant.components.light.lutron import to_hass_level
+from homeassistant.components.light.lutron import to_lutron_level
+
+REQUIREMENTS = ['https://github.com/gurumitts/'
+                'pylutron-caseta/archive/v0.2.0.zip#'
+                'pylutron-caseta==v0.2.0', 'paramiko==2.1.2']
+
+_LOGGER = logging.getLogger(__name__)
+
+
```

Comment:
@pvizeli @cgarwood,  I agreed with @cgarwood - this would be better suited as a component as the smartbridge hub does support devices beyond just the light (dimmer).  Let me get the switches also working and I will push additional changes.  I don't own any other caseta devices so I will need to code this at my brothers house.  Take another look at the PR after that and we can decide which way to proceed.  I will try to do it soon and add the config validation as well.